### DISCUSSION TITLE
Flow changes in Admin dashboard

### DISCUSSION THREAD

**Discussion by fakhruddin-vsbizz** [priyadarshi0811/arabic-moallema-refactored-v2.0#57] [Announcements]
When a batch is created, the teacher is already assigned. Thus, remove the batch addition part from the teacher details page.

**Comment by fakhruddin-vsbizz** [comment]
Admin: Assignments - routes are ready for showing assignments for a module with a submodule. Linking needs to be done.

**Comment by fakhruddin-vsbizz** [comment]
Admin: Create Assignments 
- All alphabets in Alphabet module need to be listed in the Submodule dropdown
- DragDropActivity - title should be (1) Enter the bucket list and below, (2) Enter the dragging words

**Reply by somesh2001** [reply]
first part Submodule dropdown - need to done by somesh

**Reply by somesh2001** [reply]
The first, part is **done** getting all the sub-modules apart from the sub-module whose assignment is already created.

![image](https://user-images.githubusercontent.com/56119880/225883707-3756b653-59bc-4d28-9078-ecea106508b4.png)

**Reply by atif-vsbizz** [reply]
Done With UI and Text.

![2023-03-17_180521](https://user-images.githubusercontent.com/124858584/225906197-af08c987-4ba0-44e7-8772-4104c226a1ff.png)

**Comment by fakhruddin-vsbizz** [comment]
Admin: logout - Should redirect to login page

**Reply by somesh2001** [reply]
Done - Set the route with window.location.href

![image](https://user-images.githubusercontent.com/56119880/225852756-2afe462e-75d4-4765-8887-6e6328de8928.png)

**Comment by fakhruddin-vsbizz** [comment]
Admin: Create Teacher - When creating a teacher, email should contain _As A Teacher_ instead of _As a Student_

**Reply by somesh2001** [reply]
Done

**Comment by fakhruddin-vsbizz** [comment]
Admin: Teacher Details - When the teacher or student email needs to be changed, the previous email is mailed and new email is added and replaced in the authentication.

**Reply by fakhruddin-vsbizz** [reply]
@somesh2001 can look into this
